### PR TITLE
Package ocl-icd

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

**Comment by inducer** [comment]
Close/reopen to remedy what looks like a transient CircleCI failure.

**Comment by inducer** [comment]
Ready to review from my end.

File: recipes/ocl-icd/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+set -e
+
+./configure --prefix=$PREFIX --enable-official-khronos-headers
+make
+make install
```
Comment (183amir) [inline recipes/ocl-icd/build.sh]: Is there a `make test` that you can run?

File: recipes/ocl-icd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://forge.imag.fr/frs/download.php/716/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+  # Makes ocl-icd install the OpenCL headers it ships
+  - install-headers.patch
+  - install-headers-post-automake.patch
+  - define-cl2-prefix-suffix-on-apple.patch
```
Comment (183amir) [inline recipes/ocl-icd/meta.yaml]: Do you have some references for these patches?

File: recipes/ocl-icd/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+build:
+  number: 0
+  skip: True  # [not linux or not py35]
```
Comment (183amir) [inline recipes/ocl-icd/meta.yaml]: Maybe `not (linux and py35)` would be easier to read?

File: recipes/ocl-icd/meta.yaml (hunk 2 above)
Comment (inducer) [inline recipes/ocl-icd/meta.yaml]: No, I wrote them. I chose the names to be descriptive, so a one-liner description wouldn't necessarily add value. But I'm happy to explain what I did.

**Comment by 183amir** [comment]
Since this is complicated, I'll let somebody else review it too.

File: recipes/ocl-icd/meta.yaml (hunk 3 above)
Comment (inducer) [inline recipes/ocl-icd/meta.yaml]: Good idea, done.

File: recipes/ocl-icd/build.sh (hunk 1 above)
Comment (inducer) [inline recipes/ocl-icd/build.sh]: Good suggestion, done.

**Comment by scopatz** [comment]
Looks good to me, thanks!

**Pull request merged**